### PR TITLE
optimize oracle log

### PR DESCRIPTION
**Pull request by Ashuaidehao**

### CODE REVIEW

**Review by shargon — APPROVED** [review]

**Review by Liaojinghui — APPROVED** [review]

File: src/OracleService/OracleService.cs [hunk 1]
```diff
@@ -268,12 +268,14 @@ private async Task SendResponseSignatureAsync(ulong requestId, byte[] txSign, Ke
 
         private async Task ProcessRequestAsync(DataCache snapshot, OracleRequest req)
         {
-            Log($"Process oracle request txid: {req.OriginalTxid}, url: {req.Url}");
+            Log($"[{req.OriginalTxid}] Process oracle request start:<{req.Url}>");
```
Comment (vncoelho) [inline src/OracleService/OracleService.cs:271]: begin and end

start and finish

maybe begin is better

File: src/OracleService/OracleService.cs [hunk 2]
```diff
@@ -268,12 +268,14 @@ private async Task SendResponseSignatureAsync(ulong requestId, byte[] txSign, Ke
...
 
             uint height = NativeContract.Ledger.CurrentIndex(snapshot) + 1;
 
             (OracleResponseCode code, string data) = await ProcessUrlAsync(req.Url);
 
+            Log($"[{req.OriginalTxid}] Process oracle request end:<{req.Url}>, responseCode:{code}, response:{data}");
```
Comment (vncoelho) [inline src/OracleService/OracleService.cs:277]: maybe finish

**Review by vncoelho — COMMENTED** [review]
I like the logs, I will soon test it on our examples.

**Review by superboyiii — APPROVED** [review]

**Comment by superboyiii** [comment]
Tested OK.

**Pull request merged**